src/app/shared/components/product-item/product-item-short/product-item-short.component.html

```diff
@@ -0,0 +1,42 @@
+<div *ngIf="product">
+  <div *ngIf="showShort" class="short-version" (mouseover)="isOutOfStock">
```

Comment:
mouseover handler called isOutOfStock, really? rename to onMouseOver.
isOutOfStock should be used for variable/property name and NOT for the function